volumes/vds/readme.md

```diff
@@ -0,0 +1,255 @@
+# Vehicle Detector System (VDS) data  
+
+VDS data is pulled daily from ITS Central database on terminal server by dags running on Morbius.  
```

Comment:
I think the ITSC database is accessible through the terminal server or any other on-prem server but not on the terminal server.